pkg/runner/runner.go

```diff
@@ -101,6 +87,11 @@ func Run(cfg *Config) error {
 
 	defer client.Close()
 
+	if cfg.StartupOnly {
+		log.Info("msg", "Promscale in startup-only mode, exiting post startup...")
```

Comment:
We should give more context here. As a user, I am clueless which `startup` we are talking about.

Something like, `Promscale CLI mode`